packages/web-app/src/modules/earn-views-mobile/components/MobileEarningSummary.tsx

```diff
@@ -107,8 +146,21 @@ const _MobileEarningSummary = ({
         />
       </div>
       <Divider />
-      <SectionHeader>Earning History</SectionHeader>
-      <EarningHistory viewLast24Hours={viewLast24Hours} viewLast7Days={viewLast7Days} viewLast30Days={viewLast30Days} />
+      <AllMachines
+        machines={generatedMockedMachines}
+        onMachineIdClick={setShownModalMachineId}
+        onSelectedMachineIdsChange={handleSelectedMachineIdsChange}
+      />
+      <EarningHistory
+        daysShowing={daysShowing}
+        earningPerSelectedMachines={earningPerSelectedMachines}
```

Comment:
let's changes naming 
for `EarningHistory`  component it's just machines earning, it shouldn't know they are selected
```suggestion
        machinesEarningById={machinesEarningById}
```

Reply:
Let's keep `earningsPerMachine` naming if we don't want to specify that they are selected